ports/soh2/Ship of Harkinian 2.sh

```diff
@@ -90,11 +123,11 @@ fi
 
 # Run the game
 echo "Loading, please wait... (might take a while!)" > $CUR_TTY
-$GPTOKEYB "2s2h.elf" -c "soh2.gptk" &
-pm_platform_helper 2s2h.elf
+$GPTOKEYB "2s2h.elf" -c "soh2.gptk" & 
 ./2s2h.elf
 
 # Cleanup
 rm -rf "$GAMEDIR/logs/"
-
-pm_finish
+$ESUDO systemctl restart oga_events & 
+printf "\033c" >> /dev/tty1
+printf "\033c" > /dev/tty0
```

Comment:
Same remark, just tell me if I'm wrong :)
```suggestion
pm_finish
```